build/npm/postinstall.js

```diff
@@ -23,9 +23,14 @@ function yarnInstall(location, opts) {
 	const args = original.filter(arg => arg === '--ignore-optional' || arg === '--frozen-lockfile');
 
 	console.log(`Installing dependencies in ${location}...`);
+	if (location.startsWith("remote") && process.env["npm_config_arch"] == "arm64") {
```

Comment:
Just move this further down to when we call `yarnInstall('remote')` and add the option as an argument.

Reply:
This has been replaced with a new method which applies the the override for `remote/` portions.